### ISSUE TITLE
Per layer learning_rate multiplier for fine tuning

### ISSUE THREAD

**Issue by nikolaypavlov** [Lasagne/Lasagne#482]
Hi, I would like to fine tune GoogleLenet from the model zoo, but with different data and classification layers. Is there anything similar to caffe's lr_mult parameter to prevent pre trained layers from changing to much?

**Comment by ebenolson** [comment]
No, but you can implement it yourself with something like this:

    lr_mult = {}
    lr_mult['loss3/classifier'] = 0.1
    
    DEFAULT_LR = 0.001

    updates = {}

    for name, layer in net.items():
        layer_params = layer.get_params(trainable=True)
        layer_lr = lr_mult.get(name, 1.0) * DEFAULT_LR
        layer_updates = lasagne.updates.sgd(loss, layer_params, learning_rate=layer_lr)
        updates.update(layer_updates)

**Comment by nikolaypavlov** [comment]
Thanks. That was helpful.
